pkg/cmd/get/cluster/exec.go

```diff
@@ -50,8 +66,32 @@ func (o *Options) run() (err error) {
 	}
 
 	table := converToTable(clusters)
+	tree := convertToTree(clusters, o.printer.Tree())
+	clusters.GetObjectKind().SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   clusterGroup,
+		Version: clusterAPIVersion,
+		Kind:    clusterKind,
+	})
+
+	o.printer.WithTree(tree).WithTable(table).WithYaml(clusters)
 
-	return o.printer.PrintObj(table, o.Streams.Out)
+	return o.printer.Print(o.Streams)
```

Comment:
can we have a func

```
func converToTree(obj runtime.Object, tree *printer.TreePrinter ) *printer.TreePrinter

func converToTable(obj runtime.Object) *metav1.Table
```

so when call printers, we have

```
o.printer.WithTreeConverter(convertToTree).WithTableConverter(convertToTable)
```

so when you print, you just can call

```
o.printer.Print(o.Streams, clusters)
```